### ISSUE TITLE
ATECC608A Sleep and reset the registers

### ISSUE THREAD

**Issue by gskrish7** [MicrochipTech/cryptoauthlib#324]
I am working on embedded Linux, - I am trying implement my own to sleep function., The datasheet suggests sending 0x01 to sleep the device and this will clear the registers in chip. I tried to do that, but still the chip is not going to sleep state and still taking commands without need to send wakeup signal.

As i wokrk with ioctl I configure ioctl as below
1. ioctl( fd, I2C_SLAVE, 0x36 );
2. send( fd, sleep, 1 );

I onbserved even after implementing sleep as mentioned above, the ATECC608a is accepting commands without need to send wake-up signal again. The only after introducing the sleep of 2 sec, the device goes into sleep and we wakeup and run command.

Basically the requirement is to reset registers and state of chip before executing next command.
